crates/bevy_pbr/src/lib.rs

```diff
@@ -234,9 +234,7 @@ impl Plugin for PbrPlugin {
 
         app.register_asset_reflect::<StandardMaterial>()
             .register_type::<AmbientLight>()
-            .register_type::<Cascade>()
             .register_type::<CascadeShadowConfig>()
-            .register_type::<Cascades>()
             .register_type::<CascadesVisibleEntities>()
             .register_type::<ClusterConfig>()
             .register_type::<ClusterFarZMode>()
```

Comment:
Can't we remove this? After all, `ClusterZConfig` has this types in fields.
`ClusterConfig` has some types too.

Reply:
Actually, this file has some other kinds that can be removed as well.

Reply:
Should be more cleaned up now.